src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -100,35 +99,46 @@ class BaseOptionsSelector extends Component {
     }
 
     componentDidMount() {
-        this.subscribeToKeyboardShortcut();
+        this.focusListener = this.props.navigation.addListener('focus', () => {
+            if ([CONST.PLATFORM.DESKTOP, CONST.PLATFORM.WEB].includes(getPlatform())) {
+                this.subscribeToKeyboardShortcut();
+            }
 
-        if (this.props.isFocused && this.props.autoFocus && this.textInput) {
-            this.focusTimeout = setTimeout(() => {
-                this.textInput.focus();
-            }, CONST.ANIMATED_TRANSITION);
-        }
+            if (this.props.autoFocus && this.textInput) {
+                this.focusTimeout = setTimeout(() => {
+                    this.textInput.focus();
+                }, CONST.ANIMATED_TRANSITION);
+            }
 
-        this.scrollToIndex(this.props.selectedOptions.length ? 0 : this.state.focusedIndex, false);
-    }
+            this.isFocused = true;
+        });
 
-    componentDidUpdate(prevProps, prevState) {
-        if (prevProps.isFocused !== this.props.isFocused) {
-            if (this.props.isFocused) {
-                this.subscribeToKeyboardShortcut();
-            } else {
+        this.blurListener = this.props.navigation.addListener('blur', () => {
+            if ([CONST.PLATFORM.DESKTOP, CONST.PLATFORM.WEB].includes(getPlatform())) {
                 this.unSubscribeFromKeyboardShortcut();
             }
-        }
+            this.isFocused = false;
+        });
+        this.scrollToIndex(this.props.selectedOptions.length ? 0 : this.state.focusedIndex, false);
 
-        // Screen coming back into focus, for example
-        // when doing Cmd+Shift+K, then Cmd+K, then Cmd+Shift+K.
-        // Only applies to platforms that support keyboard shortcuts
```

Comment:
Please restore this comment

Reply:
Is it important? Since the code was refactored for which the comment was relevant.

Reply:
Just to avoid breaking this when someone updates code around here